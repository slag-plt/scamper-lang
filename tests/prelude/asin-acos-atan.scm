(asin 0.4201670368266409)
(asin 0.6518337710215366)
(asin -0.8645514486106083)
(asin -0.7568024953079282)
(asin -0.18216250427209588)
(asin 0.9589242746631385)
(asin -0.6365371822219679)
(asin 0.8600694058124532)
(asin 0)
(acos 0.9074467814501962)
(acos 0.7583618759905082)
(acos -0.5025443191453852)
(acos -0.6536436208636119)
(acos 0.9832684384425845)
(acos 0.28366218546322625)
(acos 0.7712460149971067)
(acos 0.5101770449416689)
(acos 1)
(atan 0.4630211329364896)
(atan 0.8595286652169407)
(atan 1.7203486651303583)
(atan 1.1578212823495777)
(atan -0.18526223068913525)
(atan 3.380515006246586)
(atan -0.8253361052690248)
(atan 1.6858253705060158)
(atan 0)