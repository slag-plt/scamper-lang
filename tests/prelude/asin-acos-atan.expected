0.43362938564082704
0.71
-1.0442571243572367
-0.8584073464102067
-0.1831853071795868
1.2831853071795865
-0.69
1.0354056994857892
0
0.4336293856408271
0.71
2.0973355292325566
2.2831853071795867
0.18318530717958706
1.2831853071795865
0.69
1.0354056994857892
0
0.43362938564082704
0.71
1.0442571243572367
0.8584073464102068
-0.1831853071795868
1.2831853071795865
-0.69
1.0354056994857894
0
